modules/ai/assets/js/editor/pages/form-layout/components/attachments/thumbnail.js

```diff
@@ -37,7 +37,7 @@ export const Thumbnail = ( props ) => {
 			previewRef.current.style.width = `${ THUMBNAIL_SIZE }px`;
 			previewRef.current.style.height = `${ THUMBNAIL_SIZE }px`;
 		}
-	}, [] );
+	}, [ previewRef.current ] );
```

Comment:
not sure how it works when a ref change doesn't trigger a re-render
why is it needed?

Reply:
The inline-style that was added here is gone on a new render. So, we need the effect to run again. 
The `current` is changed on a re-render.